src/main.py

```
@@ -2,7 +2,9 @@
 import json
 import threading
 
-import discord
+import discord 
+from discord import Option
```

Ruff (0.8.2)

`discord.Option` imported but unused

Remove unused import: `discord.Option`

(F401)